app/controllers/offers_controller.rb

```diff
@@ -78,25 +82,24 @@ def combine_contracts_print
     end
   end
 
-  def get_contract
-    offer = Offer.find(params[:offer_id])
-    generator = ContractGenerator.new([offer.format])
-    send_data generator.render, filename: "contract.pdf", disposition: "inline"
+  def get_contract_mangled
```

Comment:
you call this get_contract_pdf 
(I can tell it's a "helper". name wants to give me a hint as to what it's helping with!!)